openeo/Sentinel-3.ipynb

```diff
@@ -0,0 +1,444 @@
fj5UInri/wxVioYhvX+YlknyOBW+POy4ahqmGjDy+tojAm9uPDlTpbc/r4Yx8ZR9prLsXWs5DZHjOFaLgMoytA4xVv0GYyQXaH3D7W5XATA92401NNZgSuHNYUdn7cLM80Zl1wW4aVt2vsE5y85ZjlPEWsNpCOy84+Ad1ggPwwglU7L6l/apsGtbQsp0TYN3nnNUiaIDfvvDe767P/I4Tvzj23tO44R3ltdXHd4ps2/ul/vW07SOXATfeL499fzdd++52nUglxAP6xXobET4+npHU5Nwv3nssaIyxvspYURDfBDD/Thx7AfeDSN9SNx2DVNNHn97GuhT5hQipykyJWVRnseAs4bb1nHbWn5xvaOIBgqdzmeysTjnudp11UcTrjtfgW0hTRPDGGmMIVa/viHqZ+aibNV3p54/3J8RCr96daNAAlRQKfAwjOSi753vx1QZYClnbg4HHvteGVipLCy1UgGgQ6ep8++PJw33qf91TcvwOQzKT90fUBdELsAjzATJZxLt+oKurazA8wo85ZKrvJsVaKcS6TKjlWXFOJxByvk1We1/+cyPAZaXhZK5/R9W3bKw2CPMT4hcjn5hewqImAq+CyVnQghMVXY8y7tn5uR6desS1nNZiFpaIWv7jNUrFZAV4Hwe+E/f3NOHwuMIEfC+4T7owsX6qz3VRbKmSoitGPbOLWdryoVzSBzrIktI9btEVDLdWIutDOebrgFRP8WUdKzpnILtqRRCZeqWVDAFUlLJuBep90RSxmeM5AriTTEqozYmhikurOrGOUT0PVIK56HHWYPzmkjvnMVax82h4/X1gWGayBiiWKZcaJznfT8SYmLnLFed529eX+NE+G9+8YZ/+8Vr/quvv+Q46qJKQjgeT8SUOPU9lksC9q+/fIWrbMvbXcPtYYcXiEXZjN6auuiTeX29xwj0IfDjeWCIiZ33pAJd4yklK7NahNedJ1dG/G/fPagtSyg01jAOE9e7jrshMI0Tf/PlgVc7x5Qzrw87ppgZIqQs7BvPzfWOVzuHrT6+ISVCTIuMfqutttpqq6222mqrrf7SawMoP1EfzGHLh8+vGUDr52ZfNZAnwQlPASmtlDPnkHjbR84hVS8y3WxhRi7kmbKaXF/Aw1nSNbMooIax5AtrBaEysqonHCrXnY9pYfSs58QLOPBzSz7y+OdWPV/yRzXiT6hLm/et5RQS745hOfd7b9m1bgVc/GnlqhmciEoBjTUcxwSVVRNTIqaokj0UhPBGUW5rjEp+s8q7Q00KVq803QeiYQ2ts0sgxRgTFsFZy8MYaKwlpMxUU59TLuycV2DQGkTgNEUNoEgZMUYl3Dmzb1v6EGmcwRlRX0SE46Aealb0noi50MdESHmRL4oIziv4GlPm0Hr1+wPdp7faZyvjeC+QEIaQiKVw3wesMRpcUsHEEAIW4dA2NckcGmeXtOVSlHkVszKZrcCUMkPMmJJBVC553VhlSBoFX0vW1G1BWVKGwjBOelwhMWEWNpg1Ct54azRFPWeGkPjfvn1bWXiFt6dzvV4K9qk/qMpk5x9v1GcyFLDW8O545HWr7C1rzXL/p1zYtQ2ncaRAlf/r8aasScP5j0rI4ana+MmtfWFwz1YXswz5KXPwAsLnrInrT8aWchnHZvBvASfnXXDZdt5keXHdvmf1/O68/P186/XYLHX8ncdguQCXsIy3C4MdldOqHDdRqrHsB8nfXLwnnzRm2WTNhl+18RnuWiq19vv7M9+8O9JPCbKGRol1i2TdyoW7KiLcdC3XXaPWEbD4wPYxEXPS+7aee28Nr/bdYvFwmgKxXjcjwpiSAmkpVrBZx41QFwGmnJhqsIvitBqAU0qhrZ6qKSXePjwSYoSsrN8xREIqamkC3O52HJqGXdvpdiXX850ZY2SoCyv9FOiniZiUPahjhq3WCZVVaUz1r0w4K7Re/y5FwcgsQkKl1MdxIqZYvVBh9h2dabTWGvbekSmErFLwUBcjUil4MdXPmQWYPY+BU4h6j+SswLJ1DOPIrnE8jgGpViLnmLk/KiO69Y7/5XdvKaILmJ13lCpnd85WabcQsMu9puxtwbntn3lbbbXVVltttdVWW/111BaS84laT0gvTz6b8BYoawn3/EaoCboz6yYvE9WcwVoNS9h5w9c7y/tBE4/XE1AjZmHpSCUiPpnwLizHy2QZhDT7ElZfQEtZko2NQLDCV60hY2mtcB9KlXi/yNuhPPvr0yft+YM/FbxbgRszGfPPWaVw7BPnMbHfCzsnpKST89dXjh8ewn+RJlkLtwdlNt4PylacorKwUs5YaxkmBeJiCJiGJSjFOE3Zvh9Vdm1FKgsqk5MCBlfeI6kgztKKcNUKMSbEwBQyf3N7zZAUoBhrQIYBHsdxYfi92nXcDyPXbcvf3F7x7vFEHyI3u5YpJh77zGmKC8imHpWOMAMZIdF5hzUKsJqSCSGRTWa363ACf/vLL8k544z27Vg0nGPf6OS/cY4pZjKRWFSeOVTQdoyRIUactViERMYYSMXwGBNW9JimGlYzpczrfcNx1OTxXePYeYfJiZ3ziAiNd/y7W+EY9XrYkmm8436Y+MXrjiFkhmK43jXEEJhCpOtaphgxxtI1jtMQiEVDLB5OI/2kEn0jwhhCBT7U36+Uwte7He9D4KZVZqp6Typo+t/95pf8599/S+csv3040loLqIR0DuEqBW52Ox1oRBhTJJbMcRyx1hCn9Lmdnycg3kqh/AREW7EYdcwsmkA/S/q57CaXsiSXL6xI4QLoMXMcL/tevf3JWLRsJ2gozhrFqy/MzM5l35XRKNWBlyd7X71nGU8vNhsL6GcNYmbvX6FUqfp8by4LUmX1CSuQawZCn6SSF5X7X04kT8b5D65MKYQp865ey29+7ClkDvuGfesYY+b1TvvhF1e7urChIWmtc/RTwBuDleoNKVCqX6sVYdd4lSGnzJR0rCkFhqAy5Z0zHKdIDuojetUoi/i+H5d2lzoWCYL3bvE4FqGC
```

Comment:
I'm not sure all this explicit code (flattening, exclude NaNs, np.histogram, ...) is necessary to just show a simple histogram, most of that will be handled automatically. 

I think something like this is just enough (starting from `data` variable):
```
data.plot.hist(bins=50, ax=axhist)
```

I also think and explicit `plt.show()` is not necessary anymore nowadays